### PR TITLE
[FIX] odoo/cli, google_gmail, microsoft_outlook: retarget doc links to 16.0

### PR DESCRIPTION
**Pull request by niyasraphy**
Description of the issue/feature this PR addresses: update url from 15 to 16

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+ squash
@fw-bot ignore

**Comment by niyasraphy** [comment]
@AntoineVDV thanks for reviewing. should i squash the commit into single one from my side and update the pr title, let me know

**Comment by AntoineVDV** [comment]
Hello @niyasraphy, yes please squash the commits if you can. I didn't see this PR had two commits :grin:

**Comment by niyasraphy** [comment]
Hi @AntoineVDV , squashed the commits into single one and updated the commit title to be same as PR title

**Comment by AntoineVDV** [comment]
Thank you!

@robodoo r+ rebase-ff

File: odoo/cli/templates/default/__manifest__.py.template [hunk 1]
```diff
@@ -14,7 +14,7 @@
     'website': "https://www.yourcompany.com",
 
     # Categories can be used to filter modules in modules listing
-    # Check https://github.com/odoo/odoo/blob/15.0/odoo/addons/base/data/ir_module_category_data.xml
+    # Check https://github.com/odoo/odoo/blob/16.0/odoo/addons/base/data/ir_module_category_data.xml
```
Comment (Julien00859) [inline odoo/cli/templates/default/__manifest__.py.template:17]: We missed the opportunity to inject the version programmatically

**Pull request closed without merging**